packages/android-client/app/build.gradle

```diff
@@ -0,0 +1,81 @@
+plugins {
```

Comment:
do we want android example? if so it should go to examples folder, right?
but it seems we don't maintain the android example anyway

Reply:
agreed, @mironiasty give us your opinion